$OpenBSD: patch-libmpeg2_motion_comp_arm_s_S,v 1.1 2018/09/16 06:08:04 ajacoutot Exp $

Set visibility of global symbols used in ARM specific assembly file to internal

Index: libmpeg2/motion_comp_arm_s.S
--- libmpeg2/motion_comp_arm_s.S.orig
+++ libmpeg2/motion_comp_arm_s.S
@@ -23,7 +23,8 @@
 
 @ ----------------------------------------------------------------
 	.align
-	.global MC_put_o_16_arm
+	.global   MC_put_o_16_arm
+	.internal MC_put_o_16_arm
 MC_put_o_16_arm:
 	@@ void func(uint8_t * dest, const uint8_t * ref, int stride, int height)
 	pld [r1]
@@ -83,7 +84,8 @@ MC_put_o_16_arm_align_jt:
 
 @ ----------------------------------------------------------------
 	.align
-	.global MC_put_o_8_arm
+	.global   MC_put_o_8_arm
+	.internal MC_put_o_8_arm
 MC_put_o_8_arm:
 	@@ void func(uint8_t * dest, const uint8_t * ref, int stride, int height)
 	pld [r1]
@@ -152,7 +154,8 @@ MC_put_o_8_arm_align_jt:
 .endm
 
 	.align
-	.global MC_put_x_16_arm
+	.global   MC_put_x_16_arm
+	.internal MC_put_x_16_arm
 MC_put_x_16_arm:
 	@@ void func(uint8_t * dest, const uint8_t * ref, int stride, int height)
 	pld [r1]
@@ -244,7 +247,8 @@ MC_put_x_16_arm_align_jt:
 
 @ ----------------------------------------------------------------
 	.align
-	.global MC_put_x_8_arm
+	.global   MC_put_x_8_arm
+	.internal MC_put_x_8_arm
 MC_put_x_8_arm:
 	@@ void func(uint8_t * dest, const uint8_t * ref, int stride, int height)
 	pld [r1]
